### ISSUE TITLE
option to see spectral profile of background corrected file in the sediment plugin

### ISSUE THREAD

**Issue by Epta13** [guiwitz/napari-sediment#7]
Can you please add the option to see the spectral profile of the bacground corrected file in the sedimen module? 
This is very much needed. Thanks

**Comment by guiwitz** [comment]
This should already work. If you import a projected with corrected data, you can go to the ```Plotting``` tab and if you hover over the image while pressing Shift it should show you spectral data for the current pixel.

**Comment by Epta13** [comment]
So it is not possible to have it available right after the background correction without exporting the data?
Do I follow the workflow correctly?

1. Load data in the Main
2. background correct in the Processing tab
3. export the data in the Export-Import tab
4. I can check the spectral profile
5. continue with the masking tools
6. create ROIs
7. once all done, export the mask and the data again?

**Comment by guiwitz** [comment]
I think at the moment the issue is that the first time the data are loaded from the original files. This doesn't allow for fast data browsing, and you need the pre-processed data in zarr format for that. What I can do is automatically reload the data after pre-processing and that should solve your problem.

**Comment by Epta13** [comment]
I tried with the shift, but the plot has wierd default axis. I would need x-axis to be wavelenth and y-axis the absorbance value. Is it possible to make this default?

**Comment by guiwitz** [comment]
This should be fixed now:
- After pre-processing, the corrected data are automatically reloaded, also no need to export anymore
- The profiles show now the correct x-scale (bands in nm)
- The profiles are now "continuum-corrected" on the x-axis

To summarize, the procedure is now:
1. Set data and export folder
2. Background correct. Note here that correcting the currently loaded bands is not sufficient. One needs to go to ```Correct full dataset```.
3. One can check per-pixel spectral profiles in the Plotting tab.
4. Create masks.
5. Create rois.
6. Export project (no need to separately export masks)

**Comment by Epta13** [comment]
Nice, thanks. First 2 points work smoothly. Only a little detail, can you remove the destripe default tick in the white correction?

Then the pixel spectral profile is better, but it shows only this:
![Screenshot 2024-02-14 101014](https://github.com/guiwitz/napari-sediment/assets/91525516/61a28248-4dbc-4198-aba7-219a8ac2a883)

Is this something in the setting of the plot?

**Comment by guiwitz** [comment]
Ah yes I can remove the destriping default. Regarding your plot: it only shows the loaded bands. So you would have e.g. to "Select all" in the main tab to load all bands. Warning: you should do this only after pre-processing, when data are efficiently loaded using zarr, otherwise it will load the entire dataset into memory.

**Comment by Epta13** [comment]
And what does the Chunk size belwo destripe does?

**Comment by guiwitz** [comment]
What do you mean with "bar"? A slider to adjust smoothening?

**Comment by Epta13** [comment]
Yes, a slider to adjust smoothening is perfect!

**Comment by guiwitz** [comment]
The two options are now available in [df0e59c](https://github.com/guiwitz/napari-sediment/commit/df0e59c2f873078f0d3c9b69f80a735cd08c1dc0). Continuum removal can be changed via a checkbox. Smoothing is available via a slider. At lowest values, not smoothing is applied. Update is reactive.

**Comment by guiwitz** [comment]
> And what does the Chunk size below destripe does?

The processed files are saved in zarr format which uses "chunking", i.e. data are split into pieces, which allows to only partially load the data. This is useful e.g. for the interactive spectral plot, which only requires to load a small area around the current pixel. I'll add some info as a tooltip.

**Comment by Epta13** [comment]
This is now working nicely. Only if the range of the x-axis wants to be changed, one must select the bands in the main menu. Maybe the range of bands can be selected also in the plotting tab?

**Comment by guiwitz** [comment]
A new issue was created to handle the new feature.
